code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -772,6 +772,12 @@ datum
 			result = "plantbgone"
 			required_reagents = list("toxin" = 1, "water" = 4)
 			result_amount = 5
+		luminol
+			name = "Luminol"
+			id = "luminol"
+			result = "luminol"
+			required_reagents = list("hydrogen" = 2, "carbon" = 2, "ammonia" = 2)

```

Comment:
Oh, that's an oversight of mine. Fixed.
Also - images! Featuring luminol, UV light and the new (old new, it was in the icons file but not an actual outfit private investigator uniform).
![ss 2015-11-30 at 01 43 48](https://cloud.githubusercontent.com/assets/7913443/11466057/7f3fe09c-9768-11e5-9a5d-88cd30ad1f78.png)